src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoInstructions.jsx

```diff
@@ -61,13 +62,10 @@ export function VideoVisitInstructions({ instructionsType }) {
         <li>
           If you’re using an iPad or iPhone for your appointment, you’ll need to
           download the{' '}
-          <a
-            target="_blank"
-            rel="noreferrer nofollow"
-            href="https://itunes.apple.com/us/app/va-video-connect/id1224250949?mt=8"
-          >
-            VA Video Connect iOS app
-          </a>{' '}
+          <NewTabAnchor
+            href="/https://itunes.apple.com/us/app/va-video-connect/id1224250949?mt=8"
+            anchorText="VA Video Connect iOS app"
```

Comment:
I'm loathe to make you update all these again, but having you pass link text through the `children` prop instead of a custom `anchorText` one would be more like normal `a` elements and less surprising.

```
<NewTabAnchor 
  href="/https://itunes.apple.com/us/app/va-video-connect/id1224250949?mt=8"
>
  VA Video Connect iOS app
</NewTabAnchor>
```

Reply:
No worries. I thought about that option as well.

Reply:
Done.